lib/const.rb

```diff
@@ -57,6 +57,26 @@
   HOME = CREW_PREFIX + Dir.home
 end
 
+CREW_IN_CONTAINER = File.exist?('/.dockerenv') || !ENV['CREW_IN_CONTAINER'].to_s.empty?
+
+CREW_CPU_VENDOR = CPUINFO['vendor_id'] || 'unknown'
+CREW_IS_AMD = CPUINFO['vendor_id'].include?('AuthenticAMD')
+CREW_IS_INTEL = CPUINFO['vendor_id'].include?('GenuineIntel')
```

Comment:
Do we need a `CREW_IS_ARM` check?

Reply:
We can grep `uname -m` for arm or aarch...

Reply:
But as each additional command slows things down, let's not do that until we need to... And we don't need an arm check at the moment...